examples/container-with-vitest/test/ReactWrapper.test.ts

```diff
@@ -1,16 +1,14 @@
 import { experimental_AstroContainer as AstroContainer } from 'astro/container';
 import { expect, test } from 'vitest';
 import ReactWrapper from '../src/components/ReactWrapper.astro';
+import { loadRenderers} from "astro:container";
+import { getContainerRenderer } from "@astrojs/react";
 
 test('ReactWrapper with react renderer', async () => {
+	const renderers = await loadRenderers([getContainerRenderer(19)])
```

Comment:
For this example, would it read better to pull this up to the top-level scope so that it's not called on every test? It's always going to be the same.